data/mods/Dark-Skies-Above/modinfo.json

```diff
@@ -2,11 +2,11 @@
   {
     "type": "MOD_INFO",
     "id": "darkskies",
-    "name": "Dark Skies Above",
-    "authors": [ "ephemeral_storyteller" ],
-    "description": "A total conversion that shifts the Cataclysm towards an alien occupation survival scenario.  THIS MOD WILL BREAK INTENDED FUNCTIONALITY OF OTHER MODS!  USE OTHER MODS AT YOUR OWN RISK.",
+    "name": "<color_light_green>Dark Skies Above</color>",
+    "maintainers": [ "Zlorthishen" ],
+    "authors": [ "Zlorthishen", "ephemeral_storyteller", "Kenan" ],
```

Comment:
Why is Kenan listed as an author here?

Reply:
because he helped

Reply:
Unmarking this as unresolved as this brings up another problem, if Kenan helped here, I see no mention of his contributions in the commit history, which is a problem.